### ISSUE TITLE
OpenGL ES 3.1+ support request

### ISSUE THREAD

**Issue by Cryolitia** [microsoft/WSA#211]
### Is your feature request related to a problem? Please describe

Some games can only run with OpenGL ES 3.1+, such as "幻塔" in China.
![image](https://user-images.githubusercontent.com/23723294/213925039-0e7f16cb-4398-4fca-b3fb-134aad610015.png)

### Describe the solution you'd like

Support OpenGL ES 3.1+

### Please specify the version of Windows Subsystem for Android

2211.40000.10.0

**Comment by celozaga** [comment]
yes, most new high fidelity mobile games already require Open GL 3.2, also it's time for WSA to support it, many emulators like Bluestacks and NetEase's MuMu are already moving towards it.

**Comment by aksj2j** [comment]
请问有解决sky在运行时提示的缺少OpenGL ES 3.1的问题吗，是否有人可以提供一下解决办法😢
